shapeshift/web
# feat: remove the cosmos plugin feature flag

## Description
It's time to remove the CosmosPlugin feature flag. It means that COSMOS assets should be findable while searching, you can access to the cosmos asset page, a cosmos account appear in the account page!

Need @0xdef1cafe in order to remove every flags from Fleek and Cloudflare


## Notice




## Pull Request Type


## Issue (if applicable)
closes #1425


## Risk
Cosmos asset is now accessible from the app, bug can be found by users


## Testing
- Build the app
- Try to find Cosmos and Cosmos on Osmosis
- The asset page should be accessible
- The account page should be accessible
- If you have some cosmos and some transactions, you should be able to see it
- The CosmosPlugin is not anymore in the feature flag


## Screenshots (if applicable)
![image](https://user-images.githubusercontent.com/14963751/162291000-884a4dd3-091e-4736-a34c-7b32cb242756.png)
![image](https://user-images.githubusercontent.com/14963751/162291043-49404aea-f1a3-4927-8f37-823a05096ef4.png)
![image](https://user-images.githubusercontent.com/14963751/162291090-61cfa0d1-ac5f-41f4-b5b9-7d067c6bf50f.png)
